_source/_posts/2018-06-05-authentication-vanilla-js.md

```diff
@@ -681,32 +686,39 @@ async deleteMeal (id) {
 
 ### Test Out Your New Vanilla JS App
 
-Congrats! You have successfully built a dynamic SPA in around 100 lines of code. Not bad! Tab back to your browser and hit refresh. You should initially see no meals as the server starts with an empty array. You should be able to add, remove, and refresh to see the app in action.
+Congrats! You have successfully built a dynamic SPA in around 100 lines of code. Not bad! Go back to your browser and hit refresh. You should initially see no meals as the server starts with an empty array. You should add meals, remove meals, and refresh the page to see the app in action.
 
 ## Add Authentication with Okta's Sign-In Widget
 
 [Okta's Sign-in Widget](https://developer.okta.com/code/javascript/okta_sign-in_widget) is a JavaScript library that gives you a full-featured and customizable login widget that can be added to any website.  With just a few lines of code, you can implement a login flow to your app.
 
 ### Configure a New Okta application
 
-To use the Okta Sign-in Widget, you must first have an Okta developer account.  If you don't have one you can [create a free account](https://developer.okta.com/signup/).  After you are logged in, click **Applications** in the top navbar and then **Add Application** button.  Make sure to select **SPA** as the platform and click **Next**.  Finally, ensure that Login redirect URIs contains `http://localhost:8080` and click **Done**.
+To use the Okta Sign-in Widget, you must first have an Okta developer account. If you don't have one you can [create a free account](https://developer.okta.com/signup/). After you are logged in, click on the **Applications** tab and then **Create App Integration** button. 
+
+Select **OIDC - OpenID Connect** -> **Single-Page Application** and click on the **Next** button. Select **Implicit (hybrid)** as the **Grant type** and un-check **Authorization Code**. Now, Enter `http://localhost:8080/` as your **Sign-in redirect URIs**. Finally, select **Allow everyone in your organization to access** as the **Controlled access** and click on the **Save** button. 
 
 Your Application settings should look similar to this:
 
 {% img blog/vanilla-js/app-settings.png alt:"SPA App Settings" width:"600" %}{: .center-image }
```

Comment:
I don't think this screenshot is necessary. We try not to include them anymore since our whole UI changed in February 2021 and we don't want to have to update all posts when the UI changes the next time. I've also noticed none of our docs include screenshots.

Reply:
Related: This post should use auth code flow with PKCE. We try not to use implicit flow except when absolutely necessary.